Webex Connect Flows/v3.0/Template/README.md

```diff
@@ -0,0 +1,50 @@
+# webexcc-digital-channels
+
+This repo provides the template workflows for Webex Contact Center and IMI integration.
+
+## Workflows Overview
+The following workflow templates are included :
+
+1. **Media Specific Workflows** : These flows need to be imported once per asset of the specific channel.
+    * **Facebook Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated Facebook page.
+    * **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured customer chat widget.
+    * **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message over the configured customer chat widget and the widget doesn't have a live chat form configured.
+    * **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms sent over the integrated SMS phone number.
+    * **Email Inbound Flow** - This workflow will be triggered for every inbound customer email to the integrated email account.
+    * **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated whatsapp business number.
+    * **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+
+
+2. **Media Agnostic Workflows** : These flows need to be imported only * once per Organization and is applicable for all channels. If v2.x shared flows are already available in
```

Comment:
Calling it as "Media Agnostic Event Workflows" make sense? 

> is applicable for all channels

by default.

Reply:
Yes it will make more sense